### PR TITLE
remove sass support

### PR DESCRIPTION
**Pull request by stoeffel**

### CODE REVIEW

File: src/ConcatModule.hs [hunk 1]
```diff
@@ -31,15 +31,13 @@ uniqNodes = LU.uniq . UT.nodesWithChildren
...
-  if compilesToJs d
-    then lift $ do
-           let name = F.pathToFileName filePath "js"
-           content <- readFile $ temp_directory </> name
-           let fnName = F.pathToFunctionName filePath "js"
-           let replacedContent = foldr replaceRequire (T.pack content) ds
-           let wrapped = wrapModule fnName replacedContent
-           return $ Just wrapped
-    else return Nothing
+  lift $ do
```
Comment (cachar) [inline src/ConcatModule.hs]: oh nice this is cleaner now 😻

File: test/ConcatModuleSpec.hs [hunk 2]
```diff
@@ -76,12 +60,9 @@ mockConfig =
     []
     ("." </> "test" </> "fixtures" </> "concat" </> "sources")
     ("." </> "test" </> "fixtures" </> "concat" </> "sources")
-    []
```
Comment (cachar) [inline test/ConcatModuleSpec.hs]: ❔ What did this `[]` do?

Comment (cachar) [reply test/ConcatModuleSpec.hs]: N/m it was the sass load paths in the old Config. I get it!

**Comment by stoeffel** [comment]
we should merge #43 first

**Pull request merged**